sdl-core/src/main/scala/io/smartdatalake/workflow/action/CustomSparkAction.scala

```diff
@@ -45,11 +47,12 @@ import org.apache.spark.sql.SparkSession
  * @param recursiveInputIds      output of action that are used as input in the same action
  * @param inputIdsToIgnoreFilter optional list of input ids to ignore filter (partition values & filter clause)
  */
+// TODO: should be renamed to CustomDataFrameAction
```

Comment:
as discussed, we will proceed with the renaming within this PR

Reply:
Done